frontend/src/services/searchfetcher/APISearchFetcher.ts

```diff
@@ -0,0 +1,20 @@
+import { Opportunity, SearchFetcher } from "./SearchFetcher";
+
+const URL = "https://jsonplaceholder.typicode.com/posts";
```

Comment:
Although it is likely just going to be a few people touching this code before it is changed to a live URL, we may want to put a generic TODO explaining that this is a placeholder URL which will be removed since the constant definition of URL is generic.

Reply:
Added a TODO, but the search page should be behind a feature flag also